src/forms/formPopup.tsx

```diff
@@ -24,6 +24,7 @@ interface FormPopupProps {
   currDatacenter?: DatacenterObject;
   initialValues?: ElementObjectType;
   elementName: ElementType;
+  loading?: boolean;
```

Comment:
this is accidentally a change meant for another branch, but not breaking since its optional